pegasus/sites.v3/code.org/views/admins_email_signup_form.haml

```diff
@@ -0,0 +1,14 @@
+%script{src: webpack_asset_path('js/code.org/views/admins_email_signup_form.js')}
+%script{charset: "utf-8", type: "text/javascript", src: "//js.hsforms.net/forms/embed/v2.js"}
+
+:javascript
```

Comment:
just out of curiousity, why can't this also be moved to the JS file?

Reply:
Oh true! I had refactored the file from using an `html` script tag to `haml` but didn't check a step further of moving that over to the new `js` file as well. I'll get on that!

Reply:
Oh actually I'm realizing I'm not sure I'll be able to import `hbspt` into the javascript file that's being retrieved in line 2